src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,52 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+import JwtService from '../services/jwt.service';
+
+const loginMiddleware = {
+  userExist: async (req, res, next) => {
```

Comment:
- Functions and methods must have a verb. E.g. `checkUserExist` and not `userExist`
- Class names must always have a noun.